config/config-example.yaml

```diff
@@ -0,0 +1,39 @@
+receivers:
+  # scrape all metrics at 1m interval
+  hostmetrics:
+    collection_interval: 1m
+    scrapers:
+      memory:
+  # add receivers to extend the ability for MySQL, JVM and Apache
+
+processors:
+  # convert from OpenTelemetry metric formats to Cloud Monitoring formats
+  metricstransform:
+    transforms:
+      - metric_name: system.memory.usage
+        action: insert
+        new_name: memory/bytes_used
+        operations:
+          # aggregate state label values: slab_reclaimable & slab_unreclaimable -> slab
+          - action: aggregate_label_values
+            label: state
+            aggregated_values: [slab_reclaimable, slab_unreclaimable]
+            new_value: slab
+            aggregation_type: sum
+
+exporters:
+  # export to Cloud Monitoring using the agent prefix
+  stackdriver:
+    project: "otel-gce-metrics" # change this to the project name of the Monitoring project
```

Comment:
Late drive-by-comment here, but we should make it so that running on GCP allows this project to be auto-detected based on the authentication credential. Having to go in and set it sort of a rough edge in the process. I think it would be worth contributing that change if needed.